config/routes.rb

```diff
@@ -1,6 +1,10 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+  get 'posts/index'
+  get 'posts/show'
```

Comment:
- [ ] We don't need to define unnecessary routes. Kindly remove these routes as you already have post show and index routes defined below.